batch.R

```diff
@@ -145,7 +145,8 @@ for(i in 1:nrow(fileDF)) {
                                      flow = qColName, dates = dateColName, time.step = "day",
                                      flow.units = getInfo(siteMeta, 'flow.units', unit.format = "rloadest"), 
                                      conc.units = getInfo(siteMeta, 'conc.units', unit.format = "rloadest"),
-                                     load.units = getInfo(siteMeta, 'load.units')))
+                                     load.units = getInfo(siteMeta, 'load.units')), 
+                             site.id = getInfo(siteMeta, 'site.id'))
```

Comment:
I added that, seemed the simplest way to fill the site.id field since it doesn't seem to really have an equivalent in the rloadest object